### PR TITLE
For Rails3 and Ruby1.9 later

### PR DESCRIPTION
**Pull request by ukstudio**
This pull-request from scaleout fork repository.
Fixed rails3 and ruby 1.9 later.

### CODE REVIEW

File: lib/acts_as_bits.rb [hunk 1]
```diff
@@ -137,8 +145,8 @@ def #{name}=(v)
               end
         end_eval
 
-        compacted_bit_names = bit_names.select{|(i,)| !i.blank?}
-        column_names = compacted_bit_names.map{|(i,)| i.to_s}
+        compacted_bit_names = bit_names.select{|(i,_)| !i.blank?}
```
Comment (hsbt) [inline lib/acts_as_bits.rb:148]: We have same fixes in our repository :octocat:

**Comment by ukstudio** [comment]
This? https://github.com/pepabo/acts_as_bits/commit/fa2d3f2cbed4197a0a1546096201fe9971360b3c

Do you have other fix commits?
If so, please merge or send pull request.

**Comment by hsbt** [comment]
@ukstudio I have no commits without https://github.com/pepabo/acts_as_bits/commit/fa2d3f2cbed4197a0a1546096201fe9971360b3c . but I'll start to migrate this plugin to rails4 env.

**Comment by ukstudio** [comment]
@hsbt :+1:

**Pull request merged**